### PR TITLE
fixes #201: allow passing custom pip options

### LINKED ISSUE #201: allow passing custom `pip install` options like `--no-index --find-links=~/wheels/`
I usually pass `--find-links=~/wheels/` to `pip install` to cache all downloaded packages. please add `--pip-options` so I can type something like this 

```
djangocms-installer --pip-options="--no-index --find-links=~/wheels/" -p my_demo -f my_demo 
```

### PR DESCRIPTION
**Pull request by muayyad-alsadi**

### CODE REVIEW

File: djangocms_installer/main.py [hunk 1]
```diff
@@ -23,9 +23,10 @@ def execute():
                              "Please wait while I install dependencies\n")
             if not config_data.no_deps:
                 if config_data.requirements_file:
-                    install.requirements(config_data.requirements_file, True)
+                    install.requirements(config_data.requirements_file, True,
+                        pip_options=config_data.pip_options)
                 else:
-                    install.requirements(config_data.requirements)
+                    install.requirements(config_data.requirements, pip_options=config_data.pip_options)
```
Comment (yakky) [inline djangocms_installer/main.py]: Please add a line break

**Comment by yakky** [comment]
@muayyad-alsadi please add the new option to the sample ini files

**Comment by muayyad-alsadi** [comment]
This one

https://github.com/nephila/djangocms-installer/blob/develop/config.ini.sample

OK should it be something that always work as blank or something that shows
a realworld usage like "--find-links=~/wheels" (or
/path/to/your/cached/wheels)

**Comment by muayyad-alsadi** [comment]
BTW I thought it would be better to swap the order instead of this

def requirements(requirements, is_file=False, pip_options='')

to this

def requirements(requirements, pip_options='', is_file=False)

So that this

install.requirements(config_data.requirements,
pip_options=config_data.pip_options)

Would become this

install.requirements(config_data.requirements, config_data.pip_options)

**Comment by yakky** [comment]
@muayyad-alsadi an empty option should do. Swapping arguments makes sense

**Comment by muayyad-alsadi** [comment]
please merge the pull and close this.

**Comment by muayyad-alsadi** [comment]
@yakky it now pass the tests please merge

**Comment by yakky** [comment]
Sorry, somehow I overlooked this ...
Merging, thanks for your contribution!

**Pull request merged**